src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -12892,7 +12896,17 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
                 case INS_umax:
                 case INS_smin:
                 case INS_smax:
-                    result.insThroughput = PERFSCORE_THROUGHPUT_2X;
+                case INS_trn1:
+                case INS_trn2:
+                    if ((ins == INS_trn1 || ins == INS_trn1)
+                        && (id->idInsOpt() == INS_OPTS_2D))
+                    {
+                        result.insThroughput = PERFSCORE_THROUGHPUT_1C;
```

Comment:
Out of curiosity - where are you getting these numbers from?

Reply:
@briansull added the PerfScore using Cortex-A55 as a base, so they can be found https://developer.arm.com/docs/epm128372/30/arm-cortex-a55-software-optimization-guide here.